### ISSUE TITLE
Bug in checkout process with Login

### ISSUE THREAD

**Issue by phillipuniverse** [BroadleafCommerce/LegacyDemoSite#9]
believe there is a bug in the demo site, i'm using the following for this scenario: http://demo35cv6c.broadleafcommerce.org/heatclinic/
1. log in as a registered user,
2. Add item A in the cart,
3. Log out.
4. Add item B in the cart and hit checkout in the cart window.
5. On the checkout Page, hit "log in to your account" under Order Information.
6. Log in using same user id as in step 1.
User is redirected to checkout page with item A appearing in the order summery and item B is lost.
Variations:
after step 3 above log in and remove item A from cart and then after step 6 in above scenario, you get empty cart warning.
Any help will be appreciated.

**Comment by dcucchietti** [comment]
Same problem here : http://forum.broadleafcommerce.org/viewtopic.php?f=15&t=1928

**Comment by phillipuniverse** [comment]
Duplicate of #11 and BroadleafCommerce/BroadleafCommerce#108
